src/Dhall/JSON.hs

```diff
@@ -192,3 +198,27 @@ omitNull (Bool bool) =
     Bool bool
 omitNull Null =
     Null
+
+{-| Convert a piece of Text carrying a Dhall inscription to an equivalent JSON Value
+
+>>> :set -XOverloadedStrings
+>>> :set -XOverloadedLists
+>>> import Dhall.Core
+>>> Dhall.JSON.codeToValue "(stdin)" "{ a = 1 }"
+>>> Object (fromList [("a",Number 1.0)])
+-}
+codeToValue :: Data.Text.Text -> Data.Text.Text -> IO Value
+codeToValue name code = do
+    let textBS = Data.ByteString.UTF8.fromString . Data.Text.unpack
```

Comment:
Oops!  Perhaps I should have suggested that the first argument have type `ByteString` that way we don't need to convert from `Text` using the `utf8-string` package

Reply:
Done.